test/e2e-tests.sh

```diff
@@ -21,7 +21,7 @@ failed=0
 
 # Run upstream knative serving operator tests
 (( !failed )) && deploy_serverless_operator_latest || failed=11
-(( !failed )) && run_knative_serving_operator_tests 'support/v0.9.0' 'cardil' || failed=12
+(( !failed )) && run_knative_serving_operator_tests 'master' || failed=12
```

Comment:
I'm happy that this is restored to master and that we no longer be using some patched version. I think that you even can omit `'master'` argument here.

Reply:
Good call, done.